onboarding/urls.py

```diff
@@ -77,8 +77,8 @@
     basename='SectionAnswers')
 router.register(r'api/user_progress/(?P<user_id>\d+)', UserProgressView,
                 basename='UserProgress')
-router.register(r'api/add_users_to_package', AddUserToPackageViewSet,
-    basename='Add user to package')
+router.register(r'api/add_users_to_package', PackagesUsersViewSet,
+                basename='Add user to package')
```

Comment:
Basename and route should be different perhaps? it's now for destroy, not only for adding.

Reply:
What would you suggest?
I didn't care about that much.
Many ideas had occurred to me, such as
`"Package users"` or `"Add user to or remove user from  package"` ...
I'm not good at this kind of thinks so :man_shrugging:

Reply:
Maybe "add_del_users_from_package" and "Add user to or remove user from the package".

BTW do you need some extra backend work on this issue, or it's already done? I can help if needed.